src/PixiEditor/Models/DocumentModels/Public/DocumentOperationsModule.cs

```diff
@@ -564,6 +564,25 @@ public void SelectionToMask(SelectionMode mode)
         Internals.ActionAccumulator.AddFinishedActions(new SelectionToMask_Action(member.GuidValue, mode));
     }
 
+    public void CropToSelection(bool clearSelection = true)
+    {
+        if (Document.SelectionPathBindable.IsEmpty)
```

Comment:
would be nice to check for TightBounds having W > 0 and H > 0 too, so that the change doesn't fail validation